### PR TITLE
feat: Cancellations consequence page

### PR DESCRIPTION
**Pull request by alainajaffer**
https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/pull/459 to be reviewed first as it branches off this

### CODE REVIEW

**Review by DeannaSharma — DISMISSED** [review]

File: site/pages/api/create-consequence-journeys.api.ts [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        if (!session) {
+            throw new Error("No session found");
+        }
+
+        if (!validatedBody.success) {
+            if (!body.disruptionId || !body.consequenceIndex) {
+                throw new Error("No disruptionId or consequenceIndex found");
+            }
+
+            setCookieOnResponseObject(
```
Comment (DeannaSharma) [inline site/pages/api/create-consequence-journeys.api.ts:82]: When I force an error (e.g. leave the consequence description blank) and try to submit the consequence, when the page loads with the error message the journeys I selected gets cleared.

Comment (alainajaffer) [reply site/pages/api/create-consequence-journeys.api.ts:82]: done

File: site/pages/create-consequence-journeys/[disruptionId]/[consequenceIndex].page.tsx
[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/pages/create-consequence-journeys/[disruptionId]/[consequenceIndex].page.tsx:1]: done

[the remark this answers is not in the document]
Comment (DeannaSharma) [reply site/pages/create-consequence-journeys/[disruptionId]/[consequenceIndex].page.tsx:1]: Changes are looking good, one thing I've noticed is that if I select a service and a journey then I go to select a different service the journeys I've selected aren't cleared. We should probably clear all journey selections when a new service is selected.

[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/pages/create-consequence-journeys/[disruptionId]/[consequenceIndex].page.tsx:1]: done

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**